x/fundraising/keeper/auction.go

```diff
@@ -186,17 +186,17 @@ func (k Keeper) DistributeSellingCoin(ctx sdk.Context, auction types.AuctionI) e
 	return nil
 }
 
-// DistributePayingCoin releases vested selling coin from the vesting reserve module account.
+// DistributePayingCoin releases the selling coin from the vesting reserve account.
 func (k Keeper) DistributePayingCoin(ctx sdk.Context, auction types.AuctionI) error {
 	for _, vq := range k.GetVestingQueuesByAuctionId(ctx, auction.GetId()) {
-		if types.IsVested(vq.GetReleaseTime(), ctx.BlockTime()) {
-			vestingReserveAcc := types.VestingReserveAcc(auction.GetId())
+		if types.IsReleased(vq.GetReleaseTime(), ctx.BlockTime()) {
+			vestingReserveAcc := auction.GetVestingReserveAddress()
```

Comment:
after checking `IsReleased` and setting `vq.Released = true` looks contradictory
This line should be refactored with `IsReleased` function name and it's logic